### DISCUSSION TITLE
考虑加入艾宾浩斯曲线或与ANKI等开源背单词软件联动

### DISCUSSION THREAD

**Discussion by Phoenix00017** [tangshimin/MuJing#45] [Ideas]
词汇累计是语言学习的基本盘，而要记住词汇，情景记忆法从理论上讲优于其他任何方式。该项目可以可以完美应用于情景记忆单词。然而，“重复”和“检测”依旧是重要的，该项目如果能够加入艾宾浩斯遗忘曲线，或与ANKI等专精于记忆的开源项目联动，那将降维打击一众商业化背单词软件。不知开发者是否有类似考虑？

**Comment by tangshimin** [comment]
有考虑过，间隔重复与现有的记忆单词的方式不兼容，需要改动的地方非常多。

如果要加间隔重复我想到两个方案:
1. 增加一个间隔重复功能，与现有的记忆方式独立，可以与现有的记忆方式快速切换，这样的话需要给现有的记忆方式取一个简单的名字，这个名字好难取啊，我想了好久都没有想到满意的，你能帮我想个我名字吗？
2. 主页界面还是现有的记忆方式， 在屏幕的左侧增加一个间隔重复开关。这样的话就不用命名现有的记忆方式了，间隔重复只是现有记忆方式的一个子功能。
